packages/reports/src/Constants/data-table-constants.tsx

```diff
@@ -405,9 +403,7 @@ export const getMultiplierOpenPositionsColumnsTemplate = ({
                     })}
                 >
                     <Money amount={Math.abs(total_profit)} currency={currency} />
-                    <div className='open-positions__profit-loss--movement'>
```

Comment:
Is it possible to get rid of this CSS in reports.scss?

Reply:
@balakrishna-deriv we are still using this class as ArrowIndicator's classname below :)